lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,32 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(outputfileloc = pot_file_path)
+      locales_path = GettextSetup.locales_path
+      output_pot = GettextSetup.config['project_name'] + '.pot'
+      if File.exist? outputfileloc
+        cmd = "mkdir -p #{locales_path}/oldpot && cp #{outputfileloc} #{locales_path}/oldpot/old_#{output_pot}"
+        _, _, _, wait = Open3.popen3(cmd)
+        exitstatus = wait.value
+        if exitstatus.success?
+          puts "Warning - #{output_pot} already exists and will be relocated to oldpot/old_#{output_pot}."
```

Comment:
If the user runs `merge-pot` once, and this happens (file already exists), and the command is run again, will there be duplicates in the resulting file? I'm not clear on how `msgcat` handles duplicates here.